backend/code_groups/cg_controllers.js

```diff
@@ -259,7 +259,7 @@ exports.updateGroupDetails = async (req, res) => {
             const userSocketId = userSocketMap.get(potentialMember.id);
             if (userSocketId) {
               try {
-                await rateLimiter.consume(potentialMember.id);
+                await rateLimiter.useRateLimiter(potentialMember.id);
```

Comment:
Looks like you're still use the package rate limiter here

Reply:
No, I'm not. I just kept the previous naming style so I wont have to change too much in my code, but rateLimiter is now referencing rate_limiter.js.

This is where it was refrenced from https://github.com/izuchukwumba/peered/pull/19/files#:~:text=const%20RateLimiter%20%3D%20require(%22./rate_limiter%22)%3B and exported here https://github.com/izuchukwumba/peered/pull/19/files#:~:text=module.exports%20%3D%20%7B%20createSocketServer%2C%20userSocketMap%2C%20rateLimiter%20%7D%3B ... So in the controller file, I am utilizing the exported rateLimiter